service/internal/logger/audit.go

```diff
@@ -0,0 +1,144 @@
+package logger
+
+import (
+	"context"
+	"encoding/json"
+	"log/slog"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type PolicyLog struct {
```

Comment:
is there a better way to do this in Go? I feel like this object type definition is disgustingly taking over half the file here. Should I move the entire `PolicyLog` definition into a new file?

Reply:
I moved them into auditPolicy.go